src/pages/slots/SlotConfiguration.tsx

```diff
@@ -1,69 +1,237 @@
                 <InputContainer>
                     <EditContainer>
                         <Label>Cupos disponibles</Label>
-                        <EditCapacity onClick={() => setShowModal(true)}>Editar
+                        <EditCapacity onClick={() => { setShowModal(true); setModalType(ModalType.EDIT); }}>Editar
                             <img
                                 src={EditIcon}
                                 width={16}
                                 height={16}
                             />
                         </EditCapacity>
                     </EditContainer>
-                    <Input disabled placeholder="15 cupos por turno" />
+                    <Input disabled placeholder="25 cupos por turno" />
                 </InputContainer>
                 <InputContainer>
                     <Label> Día del turno</Label>
-                    <Select defaultValue="">
+                    <Select onChange={handleSelectValue} value={selectEnglishValue} defaultValue="">
                         <option value="" disabled hidden>Seleccione un día</option>
                         {Object.entries(DaysOfWeek).map(([key, value]) => (
                             <option key={key} value={value}>{key}</option>
                         ))}
                     </Select>
                 </InputContainer>
-                <Button $isDisabled>Nuevo turno
+                <Button $isDisabled={selectEnglishValue === ""} disabled={selectEnglishValue === ""} onClick={() => { setShowModal(true); setModalType(ModalType.CREATE); }}>Nuevo turno
                     <img
                         src={AddIcon}
                         width={24}
                         height={24} />
                 </Button>
             </ScreenContainer>
+        )
+
+    };
+
+    const VisualizeSlotsSkeleton = () => {
+        return (
+            <SlotsContainer>
+                <TitlesContainer>
+                    <MainTitle>Turnos del {selectSpanishValue}</MainTitle>
+                    <Subtitle>
+                        {registeredSlots?.numberOfSlots}
+                        {totalSlots > 1 ? " turnos registrados" : " turno registrado"}
+                    </Subtitle>
+                </TitlesContainer>
+                {registeredSlots?.slots.map((slot, index) => (
+                    <SpecificSlotContainer
+                        key={slot.startTime}
+                        id={`slot-${slot.startTime.replace(':', '-')}`}
+                        $isLast={index === totalSlots - 1 && totalSlots > 3}
+                    >
+                        <img src={CalendarIcon} width={35} height={35}></img>
+                        <SlotInfoContainer >
+                            <SlotInfo $isBold={true}>
+                                {slot.startTime} - {slot.endTime}
+                            </SlotInfo>
+                            <SlotInfo $isDown={true}>{slot.usedCapacity}/{slot.maxCapacity} cupos ocupados</SlotInfo>
```

Comment:
No es del todo correcto tener que styled-components tan configurables porque se tienen que mostrar diferente informacion, para estos casos es mejor tener dos componentes diferentes donde cada uno de los componentes tenga los estilos que realmente necesita


```suggestion
                            <SlotTime>
                                {slot.startTime} - {slot.endTime}
                            </SlotTime>
                            <SlotCapacity>{slot.usedCapacity}/{slot.maxCapacity} cupos ocupados</SlotCapacity>
```

De esta manera no tenes que configurarlo segun que es lo que se quiere mostrar. El uso de parametros es porque quisieramos mostrar la misma informacion de distintas maneras segun la informacion que estamos mostrando, por ejemplo, el color de la letra segun el estado de la cuota. En este caso estamos cambiando el margin top, el margin bottom, el weight y tamaño de la fuente, todo segun si estamos mostrando la hora o la capacidad